composer.json

```diff
@@ -84,28 +84,19 @@
     }
   },
   "scripts": {
-    "post-install-cmd": [
```

Comment:
I'd like to keep those automated scripts so `normalize` is run on each `install/update` and we don't we don't have to worry about it. Did you encounter some problems with it?

Reply:
`composer normalize` may change `composer.json` - I leave it 100% up to you. In viktor-world this is a no-go.

Reply:
Populists do thing like _Hey computer, do the job of a human without notice_ I like to work even on tiny errors, my reward is the work!